cli/cli/out/post_process_error.go

```diff
@@ -48,9 +48,9 @@ func removeFilePathFromErrorMessage(errorMessage string) error {
 	var cleanErrorList []string
 	for _, line := range errorMessageConvertedInList {
 		// this only cleans spaces for the lines that contains the stack-trace information
-		cleanLine := strings.TrimSpace(line)
-		if !lineWithStacktrace.MatchString(cleanLine) {
-			cleanErrorList = append(cleanErrorList, cleanLine)
+		trimmedLine := strings.TrimSpace(line)
+		if !lineWithStacktrace.MatchString(trimmedLine) {
+			cleanErrorList = append(cleanErrorList, line) //
```

Comment:
trusting you on this!

Reply:
You should not! You're right I thought Caused By were indented by default but it seems they are not. Added it, good catch!